ce/ce/artifacts/artifact.ts

```diff
@@ -232,6 +234,17 @@ export class Artifact extends ArtifactBase {
       this.session.activation.addExports(exportsBlock, this.targetLocation);
     }
 
+    // if espressif install
+    if (this.metadata.info.flags.has('espidf')) {
```

Comment:
Is that get/set property earlier in the review used now? Since this is reaching into flags directly.

Reply:
Ah! no, it's not -- the espidf flag there was when we had it nested in the `git clone ` ..

I've deleted that code several times, but it keeps creeping back in when I pull from someone else 😢 

lemme remove that

Reply:
This one doesn't look changed?

Reply:
This is the correct usage, the other has been removed.